rskj-core/src/test/java/org/ethereum/TestUtils.java

```diff
@@ -227,14 +219,51 @@ public static byte[] generateBytes(long seed, int length) {
     /**
      * Generates a deterministic byte[], of requested length, for the given class and discriminator
      *
-     * @param clazz Class to generate a value for. hashCode() is applied to this parameter to obtain a seed.
+     * @param clazz         Class to generate a value for. hashCode() is applied to this parameter to obtain a seed.
      * @param discriminator String to differentiate this usage from others within the same class, method, etc. hashCode() is applied to this parameter to obtain a salt.
-     * @param length Expected result length
-     *
+     * @param length        Expected result length
      * @return the deterministic generated byte[]
      */
     public static byte[] generateBytes(Class<?> clazz, String discriminator, int length) {
         long seedWithSalt = 31L * clazz.getName().hashCode() + discriminator.hashCode();
         return generateBytes(seedWithSalt, length);
     }
+
+    public static byte[] generateBytes(String discriminator, int length) {
```

Comment:
I created this one because it was not strictly necessary to add the class to get a proper seed and it was shorter to call and to use from other classes. I think both are compatible and we could use any of them.